### ISSUE TITLE
Misleading recommendation [mb_]substr($string,0,1)

### ISSUE THREAD

**Issue by hollodotme** [kalessil/phpinspectionsea#261]
Hi, when using the following code:

```php
<?php
$string = 'Äpfel';
$first = mb_substr($string, 0, 1);
```

The inspection recommendation is:

```
$string[0] might be used instead. (invalid index accesses might show up)
```

In this case it would produce a bug, because `$string[0]  != mb_substr($string, 0, 1)`

I suggest to remove this recommendation for the `mb_substr()` function. For `substr()` it is valid.

**Comment by kalessil** [comment]
Good catch @hollodotme, will fix this.
